lib/common.js

```diff
@@ -14,6 +14,7 @@ export const supportedLocalesDetails = [
 	{ code: 'fr-ca', dir: 'ltr', name: 'Français (Canada)' },
 	{ code: 'fr-fr', dir: 'ltr', name: 'Français (France)' },
 	{ code: 'fr-on', dir: 'ltr', name: 'Français (Ontario)' },
+	{ code: 'haw-us', dir: 'ltr', name: 'ʻŌlelo Hawaiʻi (ʻAmelika Hui Pū ʻIa)' },
```

Comment:
is `-us` necessary for haw?

Reply:
Specifically, instead of `haw-haw`

Reply:
Regions must be a valid [ISO 3166](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) code. It's arguable whether using them at all for some languages is necessary, but I'm uncertain if leaving it off breaks stuff in the rest of the system. We may use it for figuring out which number system to use for example in the LMS so for now we just keep it consistent.